src/Okta.Sdk.Abstractions.UnitTests/CustomConfigurationProvidersShould.cs

```diff
@@ -88,35 +88,117 @@ public void ParseEnvironmentConfiguration()
         [Fact]
         public void ParseConfigurationObject()
         {
-            var configurationObject = new
+            // anonymous type object
+            var anonymousFilled = new
```

Comment:
I'm not clear what the intent of the suffix `Filled` is here, would `anonymousObject` communicate the intent more clearly?

Reply:
Looking at the other variable names, perhaps `anonymousComplete` might communicate the intent for this object.

Reply:
Not a blocker, I just noticed there were other object names that included `partiallySet`; this implied that the `Filled` object is completely filled.  Mostly nitpicky semantics at this point.

Reply:
I agree this name is not meaningful. What's the intent of this var?

Reply:
This var is one of the cases I want to cover:  non-typed object (anonymous, not null) with some of fields set. 
I'll think about another name for this variable.